packages/components/organisms/f-checkout/src/components/UserNote.vue

```diff
@@ -43,7 +43,7 @@ $userNote-textArea-height                        : 188px;
 
 .c-userNote {
     margin-top: spacing(x3);
-    @include font-size(body-l);
+    @include font-size(body-s);
```

Comment:
I was told just a few weeks ago that the note font size should reflect PIE and use 16px. Has this changed?

Reply:
Thought it was best to be helpful and include the text area from [Abstract here ](https://app.abstract.com/projects/da082ad2-0d14-40e8-a365-8eca471246e9/branches/master/commits/latest/files/7daa54bc-5a0f-4634-ae91-82ebae259f3c/layers/DE063997-B481-4931-9594-45EF2D4F8B1E?collectionId=f5d91e96-7233-4e1a-9800-c2065e6a3909&collectionLayerId=bb8540c5-8c8c-41a3-8257-51c61fbe3310&mode=build&selected=453671736-8569A518-E9D8-4321-A382-DA3FBF01EF35)

Reply:
Just to clarify, the label and its description should change. The placeholder and text inside the `textarea` should remain 16px.